tests/e2e-aio/e2e_test.go

```diff
@@ -259,12 +259,11 @@ var _ = Describe("E2E test", func() {
 				rerankerEndpoint := apiEndpoint + "/rerank"
 				resp, err := http.Post(rerankerEndpoint, "application/json", bytes.NewReader(serialized))
 				Expect(err).To(BeNil())
+				checkResponse(resp)
```

Comment:
this is not defined??

Reply:
my bad, during debugging was extracting into a function first.. still trying to understand why AIO tests are failing with rerankers